ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -124,38 +126,50 @@ dataFiles =
-    ,"ghc-lib/stage1/compiler/build/primop-data-decl.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-fixity.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-has-side-effects.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-list.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-out-of-line.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-primop-info.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-strictness.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-tag.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-vector-tycons.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-vector-tys-exports.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-vector-tys.hs-incl"
-    ,"ghc-lib/stage1/compiler/build/primop-vector-uniques.hs-incl"
-    ,"ghc-lib/stage0/compiler/build/Fingerprint.hs"
+    [ "ghc-lib/generated/ghcautoconf.h"
+    , "ghc-lib/generated/ghcplatform.h"
+    , "ghc-lib/generated/ghcversion.h"
+    , "ghc-lib/generated/DerivedConstants.h"
+    , "ghc-lib/generated/GHCConstantsHaskellExports.hs"
+    , "ghc-lib/generated/GHCConstantsHaskellType.hs"
+    , "ghc-lib/generated/GHCConstantsHaskellWrappers.hs"
+    , "ghc-lib/stage1/compiler/build/ghc_boot_platform.h"
+    , "ghc-lib/stage1/compiler/build/primop-can-fail.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-code-size.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-commutable.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-data-decl.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-fixity.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-has-side-effects.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-list.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-out-of-line.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-primop-info.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-strictness.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-tag.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-vector-tycons.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-vector-tys-exports.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-vector-tys.hs-incl"
+    , "ghc-lib/stage1/compiler/build/primop-vector-uniques.hs-incl"
+    , "ghc-lib/stage0/compiler/build/Fingerprint.hs"
     -- Be careful not to add these files to a ghc-lib.cabal, just
     -- ghc-lib-parser.cabal.
-    ,"ghc-lib/stage1/compiler/build/Config.hs"
-    ,"ghc-lib/stage0/compiler/build/Parser.hs"
-    ,"ghc-lib/stage0/compiler/build/Lexer.hs"
-    ,"ghc-lib/stage0/libraries/ghc-boot/build/GHC/Version.hs"
+    , "ghc-lib/stage1/compiler/build/Config.hs"
+    , "ghc-lib/stage0/compiler/build/Parser.hs"
+    , "ghc-lib/stage0/compiler/build/Lexer.hs"
+    , "ghc-lib/stage0/libraries/ghc-boot/build/GHC/Version.hs"
     ]
 
+-- | The C headers shipped with ghc-lib. These globs get glommed onto
+-- the 'extraFiles' above as 'extra-source-files'.
+cHeaders :: [String]
+cHeaders =
+  [ "includes/ghcconfig.h"
```

Comment:
I prefer commas begin lines and like in English, I believe whitespace should follow. The first line gets indented so that the `"a"` and `"b"` line up in accordance with the above.
```
[  "a"
,  "b"]
```

Reply:
Sorry should have been more clear, I was referring to the space inside the string literals in the following lines.

Reply:
Oh! That's a mistake! Raising a PR.